### DISCUSSION TITLE
why copilot labs are not working

### DISCUSSION THREAD

**Discussion by amandeepsinghslk** [community/community#80919] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I was using GitHub Copilot for language translation, but suddenly Copilot Labs is not working

**Comment by harrylal** [comment, marked as answer]
Hi @amandeepsinghslk , The copilot Labs technical preview ended on December 15th 2023.
You can Read more about it at [GitHub Next • Technical Preview Sunsets](https://gist.github.com/idan/325676d192b32f169b032fde2d866c2c#github-next--technical-preview-sunsets)
>[!TIP]
> If you’ve been loving GitHub Copilot Labs, we recommend you join the [public beta](https://docs.github.com/en/copilot/github-copilot-chat/using-github-copilot-chat) for GitHub Copilot Chat to continue enjoying this feature
